README.md

```diff
@@ -154,3 +169,5 @@ Released under [MIT license][license]
 [waffle]: https://waffle.io/GlobalNamesArchitecture/gnparser
 [waffle_progress_svg]: https://badge.waffle.io/GlobalNamesArchitecture/gnparser.svg?label=in%20progress&title=In%20Progress
 [waffle]: https://waffle.io/GlobalNamesArchitecture/gnparser
+[jruby-installation]: http://jruby.org/getting-started
```

Comment:
Do you know that a reference could be placed closer to a section? (Like I did for Jython) What do you think of that approach.